### PR TITLE
fix(zpages): use tonic based generated files.

### PR DESCRIPTION
**Pull request by TommyCpp**
related to #1208 

## Changes

The reason zpages cannot compile after #1202 is prost backed grpcio compiler no longer allow us to add `serde` macros onto the types. Thus, instead of using grpcio types, use tonic types in zpages fixes it.

## Merge requirement checklist

* [x [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-rust/blob/main/CONTRIBUTING.md) guidelines followed
* [ ] Unit tests added/updated (if applicable)
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial, user-facing changes

### CODE REVIEW

**Comment by shaun-cox** [comment]
@TommyCpp, I think if you rebase or merge, the CI/docs step will pass. (I'm eager to see this merged as soon as possible so we can merge #1199). Thanks.

**Review by shaun-cox — APPROVED** [review]

**Comment by TommyCpp** [comment]
Waiting for CI to clear and I will merge

**Pull request merged**
